Sources/Automerge/patch.swift

```diff
@@ -4,10 +4,18 @@ import enum AutomergeUniffi.PatchAction
 typealias FfiPatchAction = AutomergeUniffi.PatchAction
 typealias FfiPatch = AutomergeUniffi.Patch
 
+/// A collection of changes to an Automerge document.
 public struct Patch: Equatable {
+    /// The type change
     public let action: PatchAction
+
+    /// The path to the object identifier the action effects.
     public let path: [PathElement]
 
+    /// Creates a new patch
+    /// - Parameters:
+    ///   - action: The type of change
+    ///   - path: A collection of the paths to apply the change.
```

Comment:
The path isn't a collection of paths, it's a single path made up of `PathElement`s

Reply:
I'll update it - I was going just off the API signature that had that as a list of the type PathElement, and wasn't sure how that stitched together. That tidbit/confusion is also in my API questions list ;-) And I think you just cleared it up for me, although I might need a touch more detail about what a PathElement is, and how they're expected to be arranged and/or consumed. I'm guessing this is something a developer would more "want to react to" than construct themselves, but I wasn't sure.